test/e2e/node/mount_propagation.go

```diff
@@ -102,8 +105,8 @@ var _ = SIGDescribe("Mount propagation", func() {
 		// running in parallel.
 		hostDir := "/var/lib/kubelet/" + f.Namespace.Name
 		defer func() {
-			cleanCmd := fmt.Sprintf("sudo rm -rf %q", hostDir)
-			e2essh.IssueSSHCommand(cleanCmd, framework.TestContext.Provider, node)
+			cleanCmd := fmt.Sprintf("rm -rf %q", hostDir)
```

Comment:
For my own understanding, can you share why we no longer need to use sudo?

Reply:
we're definitely root now (the pod is root)